lib/statsig.ex

```diff
@@ -28,22 +28,20 @@ defmodule Statsig do
     get_config(user, experiment)
   end
 
-  def log_event(event) do
-    Statsig.Logging.log_event(event)
-  end
+  defdelegate log_event(event), to: Statsig.Logging
 
   def log_event(user, event_name, value, metadata) do
     user = Statsig.Utils.get_user_with_env(user)
 
     event = %{
-      :eventName => event_name,
-      :value => value,
-      :metadata => metadata,
-      :time => System.system_time(:millisecond),
-      :user => Statsig.Utils.sanitize_user(user)
+      eventName: event_name,
+      value: value,
+      metadata: metadata,
+      time: System.system_time(:millisecond),
+      user: Statsig.Utils.sanitize_user(user)
     }
 
-    Statsig.Logging.log_event(event)
+    log_event(event)
   end
 
   defdelegate flush(), to: Statsig.Logging
```

Comment:
nit: It's common practice to put defdelegates at the top of the module, before the function definitions.